### PR TITLE
KULRICE-14145 : Fix for Predicate TestProblem

### PR DESCRIPTION
**Pull request by cwade179**

### CODE REVIEW

File: rice-middleware/core/api/src/test/groovy/org/kuali/rice/core/api/criteria/PredicateTest.groovy [hunk 1]
```diff
@@ -81,9 +81,9 @@ public class PredicateTest {
 
         List<EqualPredicate> equalPredicates = foundOr.getPredicates().asList();
         // order is not guaranteed
-        if (StringUtils.equals(equalPredicates.get(0).getPropertyPath(), "name.first")) {
+        if (equalPredicates.get(0).getPropertyPath().equals("name.first")) {
```
Comment (cniesen) [inline rice-middleware/core/api/src/test/groovy/org/kuali/rice/core/api/criteria/PredicateTest.groovy]: That works.  The StringUtils import was missing.

File: rice-middleware/impl/src/test/java/org/kuali/rice/kew/rule/GenericAttributeContentTest.java [hunk 2]
```diff
@@ -86,8 +86,9 @@
         content = gac.generateContent(properties);
 
         // order is not guaranteed
-        List<String> validValues = Arrays.asList(ATTRIB2_CONTENT.replaceAll("\\s+", ""), ATTRIB2_CONTENT_REVERSED
-                .replaceAll("\\s+", ""));
-        assertTrue(validValues.contains(content.replaceAll("\\s+", "")));
+        List<String> validValues = Arrays.asList(ATTRIB2_CONTENT.replaceAll("\\s+", ""), ATTRIB2_CONTENT_REVERSED.replaceAll(
+                "\\s+", ""));
+        content = content.replaceAll("\\s+", "");
+        assertTrue(validValues.contains(content));
```
Comment (cniesen) [inline rice-middleware/impl/src/test/java/org/kuali/rice/kew/rule/GenericAttributeContentTest.java]: Just curious, did that cause an actual issue or is it just for readability?

**Comment by cniesen** [comment]
ok to test

File: rice-middleware/impl/src/test/java/org/kuali/rice/kew/rule/GenericAttributeContentTest.java (hunk 2 above)
Comment (cwade179) [inline rice-middleware/impl/src/test/java/org/kuali/rice/kew/rule/GenericAttributeContentTest.java]: partial readability and problem diagnosis.

**Comment by cniesen** [comment]
retest this please

**Comment by cniesen** [comment]
Looks good.  Check for `<<< FAILURE` in the ci job once it's done since the indicator currently isn't working right.

Note there are already some test failures so this build might not be perfect but still can be merged.

**Comment by cniesen** [comment]
I only see the JpaMetadataProviderTest
testGetMetadataForRelationship_anotherReferencedObject_main failure.  Ok to merge this commit.

**Comment by cniesen** [comment]
retest this please

**Comment by cniesen** [comment]
retest this please

**Comment by cniesen** [comment]
retest this please

**Pull request merged**